### PR TITLE
Add timeout option for uploading large builds

### PR DESCRIPTION
**Pull request by adingler6**
closes https://github.com/microsoft/fastlane-plugin-appcenter/issues/31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ The action parameters `api_token` and `owner_name` can also be omitted when thei
 - `APPCENTER_DISTRIBUTE_RELEASE_NOTES` - Release notes
 - `APPCENTER_DISTRIBUTE_RELEASE_NOTES_CLIPPING` - Clip release notes if its length is more then 5000, `true` by default
 - `APPCENTER_DISTRIBUTE_RELEASE_NOTES_LINK` - Additional release notes link
+- `APPCENTER_TIMEOUT` - Sets the request timeout in seconds. Used when uploading builds to App Center.
```
Comment (evkhramkov) [inline README.md]: let's name it `APPCENTER_DISTRIBUTE_TIMEOUT`

File: lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb [hunk 2]
```diff
@@ -412,6 +413,12 @@ def self.available_options
                                        description: "The version number. Used (and required) for uploading Android ProGuard mapping file",
                                        optional: true,
                                        type: String),
+
+          FastlaneCore::ConfigItem.new(key: :timeout,
+                                       env_name: "APPCENTER_TIMEOUT",
```
Comment (evkhramkov) [inline lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb]: same here `APPCENTER_DISTRIBUTE_TIMEOUT `

**Comment by evkhramkov** [comment]
Thank you for contribution!

**Comment by adingler6** [comment]
@evkhramkov Updated the `env_name`. Thanks for the review!

**Comment by adingler6** [comment]
@evkhramkov Any idea when this will get released? Thanks!

**Comment by edias** [comment]
Hi, @evkhramkov, do you have any idea when this one will be available?

Thank you very much for your time!

**Comment by jaleksynas** [comment]
@evkhramkov Any update on this PR, I could really use this feature to complete my appcenter migration!

**Comment by evkhramkov** [comment]
@adingler6 @jaleksynas sorry for delay, please check out [latest plugin release `1.1.1`](https://rubygems.org/gems/fastlane-plugin-appcenter/versions/1.1.1)

**Pull request merged**